src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/SWAFooter.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button, Text, Box } from "@artsy/palette"
+import { RouterLink } from "System/Router/RouterLink"
+import { useMarketingLandingTracking } from "Apps/Consign/Routes/MarketingLanding/Utils/marketingLandingTracking"
+
+export const SWAFooter: React.FC = () => {
```

Comment:
Renamed `Footer` to `SWAFooter` to not confuse it with the global footer component.